tests/mock_module/resource.py

```diff
@@ -1,10 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+
 """Example resource."""
 
-from invenio_records_resources.resources import \
-    RecordResource as RecordResourceBase
-from invenio_records_resources.resources import \
-    RecordResourceConfig as RecordResourceConfigBase
+from marshmallow import Schema
+from marshmallow_utils.fields import Link
+from uritemplate import URITemplate
 
+# TODO: Make isort NOT want imports formatted this way
```

Comment:
Comment: that will be a bit harder, and imply changes it 92 repositories if we change it ;-)

Reply:
A man can dream... :comet: 

It's ok I will remove the comment :smile: